contrib/lib/src/databases.rs

```diff
@@ -700,6 +707,27 @@ impl Poolable for memcache::Client {
     }
 }
 
+#[cfg(feature = "arango_pool")]
+impl Poolable for arangors::Connection {
+    type Manager = r2d2_arangors::pool::ArangoDBConnectionManager;
+    type Error = r2d2::Error;
+
+    fn pool(db_name: &str, rocket: &rocket::Rocket) -> PoolResult<Self> {
+        #[derive(Deserialize, Debug)]
+        struct ArangoConfig {
+            username: String,
+            password: String,
+            use_jwt: bool,
+        }
+
+        let config = Config::from(db_name, rocket)?;
+        let extra = Figment::from(rocket.figment())
+            .extract_inner::<ArangoConfig>(&format!("databases.{}", db_name))?;
```

Comment:
These can be on one line. Also, `Figment::from(rocket.figment()` is redundant; `rocket.figment()` is already a `Figment`.

Reply:
Ah...i oriented myself on what is done in `Config::from()`, but that is to please the borrow checker?

Reply:
That one calls `.merge()` on the `Figment`, which requires a move of `Figment`, but `rocket.figment()` gives you an `&Figment`. Thus, we either `.clone()` or create a new one from the borrow; I chose the latter for no particular reason. Here, you simply extract from the `Figment`, so no move is needed and you can use the borrow directly.